### PR TITLE
Fix typo in generic-interfaces.md

### PR DESCRIPTION
**Pull request by focuse**
fixed a simple typo, that is **"contravariant"** is a single word.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/programming-guide/generics/generic-interfaces.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ It is often useful to define interfaces either for generic collection classes, o
...
   
- Generic interfaces can inherit from non-generic interfaces if the generic interface is contra-variant, which means it only uses its type parameter as a return value. In the .NET Framework class library, <xref:System.Collections.Generic.IEnumerable%601> inherits from <xref:System.Collections.IEnumerable> because <xref:System.Collections.Generic.IEnumerable%601> only uses `T` in the return value of <xref:System.Collections.Generic.IEnumerable%601.GetEnumerator%2A> and in the <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property getter.  
+ Generic interfaces can inherit from non-generic interfaces if the generic interface is contrvariant, which means it only uses its type parameter as a return value. In the .NET Framework class library, <xref:System.Collections.Generic.IEnumerable%601> inherits from <xref:System.Collections.IEnumerable> because <xref:System.Collections.Generic.IEnumerable%601> only uses `T` in the return value of <xref:System.Collections.Generic.IEnumerable%601.GetEnumerator%2A> and in the <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property getter.
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/generics/generic-interfaces.md]: There's a typo in "contrvariant".

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @focuse, for contributing to the dotnet/docs repo and removing the unnecessary hyphen. However, you've also introduced a new typo. Once you fix it, we'll merge your PR.

File: docs/csharp/programming-guide/generics/generic-interfaces.md (hunk 1 above)
Comment (focuse) [reply docs/csharp/programming-guide/generics/generic-interfaces.md]: fixed `contrvariant` -> `contravariant`.

**Review by BillWagner — APPROVED** [review]
Thanks for making this fix @focuse 

@rpetrusha I've reviewed the commit after your comments, and this LGTM. I'll let you approve and :shipit:

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**